dev-libs/libx86emu/libx86emu-1.1.ebuild

```diff
@@ -10,7 +10,7 @@ HOMEPAGE="https://www.opensuse.org/"
 SRC_URI="https://download.opensuse.org/source/factory/repo/oss/suse/src/${P}-9.8.src.rpm"
 
 LICENSE="HPND"
-SLOT="0"
+SLOT="0/$(ver_cut 1-2)"
```

Comment:
This needs a new revision. Also, are you sure it's going to change on every point release too? (e.g. 1.1->1.2)?